components/Navigation/StyledNavigation.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import { IoBicycle, IoMapOutline } from "react-icons/io5";
+import { FaRoute, FaRegStar } from "react-icons/fa";
+import Link from "next/link";
+
+const Footer = styled.footer`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  height: 4rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--white);
+  background-color: var(--orange);
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-around;
+`;
+
+const NavLink = styled(Link)`
+  color: ${(props) =>
+    props.isactive === "on" ? "var(--dark-gray)" : "var(--white)"};
+`;
```

Comment:
this doesn't quite reflect in the preview of your app.

Maybe it is not the color of your Link that needs to change, but the color of the SVG?

Reply:
Moin Niklas,
your suggestion using the useRouter hook works really good. Looks so much cleaner without the handlers😍. But the highlighting of my nav bar icons works fine on my devices. I tried my iphone, edge and chrome. Can you try it again please and give me new feedback?

Reply:
Moin Kevin,

now it works for me as well 🤔 

Not really sure why it did not display properly yesterday for me. But glad that you double checked!
Then please ignore my suggestion :)

Reply:
Oh that's hard for me to do so. But I will try to ignore your suggestion this time😜
Thank's for reviewing my code again😊
I've posted my PR again. Can you give it a "weißes häkchen" please?😁